doc/source/user/absolute_beginners.rst

```diff
@@ -1864,9 +1870,7 @@ All you need to do to plot your values is run::
   >>> .plot(a) 
   >>> plt.show()
 
-.. plot:: plots/matplotlib1.py
```

Comment:
looking at the CircleCI traceback, this will work if changed to
```
.. plot:: user/plots/matplotlib1.py
```

Reply:
Thank you!